waiter/src/waiter/scaling.clj

```diff
@@ -243,10 +245,22 @@
                               (assoc executor-state :last-scale-down-time (t/now))
                               executor-state)
                             (do
-                              (log/debug "skipping scale-down as" inter-kill-request-wait-time-ms "ms has not elapsed since last scale down operation")
+                              (log/debug "skipping scale-down as" inter-kill-request-wait-time-ms
+                                         "ms has not elapsed since last scale down operation")
                               (counters/inc! (metrics/service-counter service-id "scaling" "scale-down" "ignore"))
                               executor-state)))
 
+                        (and scale-to-instances
+                             task-count
+                             (neg? scale-amount)
+                             (neg? (- task-count scale-to-instances)))
```

Comment:
Would this be more simply stated as `(< task-count scale-to-instances)`?

Reply:
Done.